docker/Dockerfile

```diff
@@ -6,21 +6,23 @@ ARG APT_MIRROR="mirror://mirrors.ubuntu.com/mirrors.txt"
 # ARG APT_MIRROR="http://mirror.aarnet.edu.au/pub/ubuntu/archive/"
 
 # Configure mirror. Pass --build-arg APT_MIRROR=<mirror URL> to set a mirror if this is slow
-RUN sed -i "s#htt[p|ps]://archive.ubuntu.com/ubuntu/#$APT_MIRROR#g" /etc/apt/sources.list
+RUN sed -i "s#htt[p|ps]://archive.ubuntu.com/ubuntu/#$APT_MIRROR#g" /etc/apt/sources.list.d/ubuntu.sources
 
 # Install packages
 RUN \
     apt update && \
     apt install -y git python3 && \
-    apt install -y cmake gcc-arm-none-eabi libnewlib-arm-none-eabi build-essential
+    # For Pico SDK
+    apt install -y cmake gcc-arm-none-eabi libnewlib-arm-none-eabi libstdc++-arm-none-eabi-newlib build-essential
 
 # Install Pico SDK into a new stage
 FROM base as buildtools
```

Comment:
docker complains that the command words to not match case (ie it wants `FROM base AS buildtools`)